conda_smithy/lint_recipe.py

```diff
@@ -261,6 +261,14 @@ def run_conda_forge_lints(meta, recipe_dir, lints):
         if feedstock_exists:
             lints.append('Feedstock with the same name exists in conda-forge')
 
+        bio = gh.get_user('bioconda').get_repo('bioconda-recipes')
+        try:
+            bio.get_dir_contents('recipes/{}'.format(recipe_name))
+        except github.UnknownObjectException as e:
+            pass
+        else:
+            lints.append('Recipe with the same name exists in bioconda')
```

Comment:
Can you ping `@conda-forge/bioconda-recipes` here?

Reply:
Sorry, I went on vacation and lost track of this. Updated now.

Reply:
Sorry, I went on vacation and lost track of this. Updated (and rebased to latest `master`) now.